Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -283,6 +302,92 @@ namespace AzToolsFramework::Prefab
+        AZ_Assert(owningInstance != AZStd::nullopt, "Owning Instance is null");
+
+        // Retrieve the path from the focused prefab instance to the owningInstance of the entityId.
+        AZStd::string prefix;
+        InstanceOptionalReference focusedPrefabInstance = GetInstanceReference(m_rootAliasFocusPath);
+
+        {
+            // Climb up the instance hierarchy from this instance until you hit the focused prefab instance.
+            InstanceOptionalConstReference instance = owningInstance;
+            AZStd::vector<InstanceOptionalConstReference> instancePath;
+
+            while (&instance->get() != &focusedPrefabInstance->get() && instance.has_value())
+            {
+                instancePath.emplace_back(instance);
+                instance = instance->get().GetParentInstance();
+            }
+
+            if (&instance->get() != &focusedPrefabInstance->get())
+            {
+                AZ_Error(
+                    "Prefab", false,
+                    "AppendPathFromFocusedInstanceToPatchPaths - entityId is not owned by a descendant of the focused prefab instance.");
+                return linkId;
+            }
+
+            for (auto instanceIter = ++instancePath.rbegin(); instanceIter != instancePath.rend(); ++instanceIter)
+            {
+                prefix.append("/Instances/");
+                prefix.append((*instanceIter)->get().GetInstanceAlias());
+            }
+
+            if (auto size = instancePath.size(); size > 0)
+            {
+                linkId = instancePath[size - 1]->get().GetLinkId();
+            }
+        }
+
+        prefix.append("/");
+
+        bool isContainerEntity = entityId == owningInstance->get().GetContainerEntityId();
+        if (isContainerEntity)
+        {
+            prefix += PrefabDomUtils::ContainerEntityName;
+        }
+        else
+        {
+            EntityAliasOptionalReference entityAliasRef = owningInstance->get().GetEntityAlias(entityId);
+            prefix += PrefabDomUtils::EntitiesName;
+            prefix += "/";
+            prefix += entityAliasRef->get();
+        }
+
+        // update all entities or just the single container
+        for (auto& patchValue : providedPatch.GetArray())
+        {
+            auto pathIter = patchValue.FindMember("path");
+
+            if (pathIter == patchValue.MemberEnd() || !(pathIter->value.IsString()))
+            {
+                AZ_Error(
+                    "Prefab", false,
+                    "Was not able to find path member within patch dom. "
+                    "A non prefab dom patch may have been passed in.");
+                continue;
+            }
+
+            AZStd::string path = prefix + pathIter->value.GetString();
+
+            pathIter->value.SetString(path.c_str(), static_cast<rapidjson::SizeType>(path.length()), providedPatch.GetAllocator());
+        }
```

Comment:
You can reuse InstanceToTemplatePropagator::AppendEntityAliasToPatchPaths() for this